### ISSUE TITLE
Ensure backup admins have access to solr and fedora

### ISSUE THREAD

**Issue by mark-dce** [curationexperts/laevigata#688]
**ISSUE**
It looks like I have named access (mark@) and deploy@ access to the apache servers in each Emory cluster, but not to the fedora and solr servers in each cluster.

**ACCEPTANCE**
- [ ] Ensure at least one other DCE user has access to all Emory servers - i.e. add their public keys 
- [x] ~~Document the connection command if it's different from `ssh deploy@host_IP` or `ssh host_IP`~~ *not different*

**NOTES**
* I seems like it might be good to make sure  at least Alicia, Mark, and Bess all have ssh access to all servers.

* We should decide if other team-members should have deploy access to the production environment or not (I'd lean toward granting it to keep config scripting identical)

* We should make sure that all development team members have the necessary access to deploy to qa, stage, and etd1 (emory.curationexperts.com) - I think this is already the case, but am writing it down to double check.

* I set up an access tracking/testing spreadsheet here if it's helpful - it shows the ssh commands I tried using and whether they were successful or not https://docs.google.com/spreadsheets/d/1CBblDacKlyjVPZF80pDkZlaOQKqE_hplIHFJkmZaAec/edit#gid=0

**Comment by mark-dce** [comment]
I retested and I have access to all servers in the qa, stage, and prod clusters.  It looks like I'm still missing access to etd1-solr and etd1-fredora or I'm doing something wrong trying to connect to them (but I'm trying the same thing that works in all the other cases).

I also updated the spreadsheet https://docs.google.com/spreadsheets/d/1CBblDacKlyjVPZF80pDkZlaOQKqE_hplIHFJkmZaAec/edit#gid=0

I took the lines about deploy user access to solr and fedora servers out since that doesn't really make much sense on reflection.
